internal/core/src/index/knowhere/external/boost_ext/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+find_package(Boost REQUIRED)
```

Comment:
You already copied a  boost_ext at `thirdparty/`, this one is dup.

Reply:
- [ ] easyloggingpp and nlohmann should be in `thirdparty/`

- [ ] Please delete `external/`